### PR TITLE
Assignment5

### PR DESCRIPTION
**Pull request by meta-nilesh-patel**

### CODE REVIEW

File: SCF/Assignment5C/NQueenProblem.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    } 
+  
+    /**
+     * 
+     * A recursive function to solve N Queen problem at initial startColumn value should be 0.
+     * @startColumn : in which startColumn we are placing queen.
+     */ 
+    boolean solveNQueen(int[][]board, int startColumn, final int noOfQueens) throws AssertionError 
+    { 
+    	if(noOfQueens < 1)
```
Comment (rash-tiwari) [inline SCF/Assignment5C/NQueenProblem.java]: Add ```tab``` rather than space

File: SCF/Assignment5A/LcmHcf.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	}
+	
+	/**
+	 * 
+	 * This method will compute LCM of two given numbers.
+	 * @param firstNumber : first number for compute LCM and it should be greater than zero.
+	 * @param secondNumber : second number for compute LCM and it should be greater than zero.
+	 * @return LCM of firstNumber and secondNumber.
+	 */
+	public int getLCM(int firstNumber, int secondNumber) throws AssertionError
```
Comment (rash-tiwari) [inline SCF/Assignment5A/LcmHcf.java]: In my opinion we have to write a logic here to get LCM using ```Recursion```

File: SCF/Assignment5C/NQueenProblem.java [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+     } 
+  
+     /**
+      * 
+      * A recursive function to solve N Queen problem at initial startColumn value should be 0.
+      * @startColumn : in which startColumn we are placing queen.
+      */ 
+     boolean solveNQueen(int[][]board, int startColumn, final int noOfQueens) throws AssertionError 
+     { 
+          if(noOfQueens < 1)
```
Comment (rash-tiwari) [inline SCF/Assignment5C/NQueenProblem.java:61]: Correct the formatting

**Review by rash-tiwari — CHANGES_REQUESTED** [review]
First Level code review done.

File: SCF/Assignment5C/NQueenProblem.java (hunk 1 above)
Comment (meta-nilesh-patel) [reply SCF/Assignment5C/NQueenProblem.java]: Done

File: SCF/Assignment5A/LcmHcf.java (hunk 2 above)
Comment (meta-nilesh-patel) [reply SCF/Assignment5A/LcmHcf.java]: Done

File: SCF/Assignment5C/NQueenProblem.java (hunk 3 above)
Comment (meta-nilesh-patel) [reply SCF/Assignment5C/NQueenProblem.java:61]: Done

**Review by rash-tiwari — APPROVED** [review]

**Pull request merged**